src/main/java/com/extendedclip/papi/expansion/player/PlayerExpansion.java

```diff
@@ -173,6 +175,8 @@ public String onRequest(OfflinePlayer player, String identifier) {
                 return bool(p.getInventory().firstEmpty() > - 1);
             case "server":
             case "servername":
+            	if(serverVersion >= 14) return "";
```

Comment:
not return ""
need to be change to `Bukkit.getServer().getName()`, https://github.com/PlaceholderAPI/Player-Expansion/commit/c73bf086e1f3160a21e37ede97035c1efb586384#r33377977

Reply:
Server#getName only represents the server implementation in 1.14. It would give back "CraftBukkit" every time. Is that what we want?